extensions/amazon-lambda/runtime/src/main/java/io/quarkus/amazon/lambda/runtime/PreInitRecorder.java

```diff
@@ -0,0 +1,50 @@
+package io.quarkus.amazon.lambda.runtime;
+
+import java.io.BufferedInputStream;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.nio.charset.StandardCharsets;
+
+import io.quarkus.runtime.annotations.Recorder;
+
+/**
+ * The class pre-initialization recorder.
+ */
+@Recorder
+public final class PreInitRecorder {
```

Comment:
Recorders can't be `final`

Reply:
I opened https://github.com/quarkusio/quarkus/pull/29024 to ensure that this mistake is caught at extension build time

Reply:
I thought I pushed up a fix for this but I guess it didn't go. Will double check in a minute.

Reply:
Ah my push was predicated on tests passing but I get some weird failure on something called "quackiverse" (SIC). Some kind of duck-related test?

Reply:
LOL